dependabot.yml

```diff
@@ -0,0 +1,20 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "DevOps-MBSE/aac-program-team"
+    labels:
+      - "dependencies"
+  
+  - package-ecosystem: "pip"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+    reviewers:
```

Comment:
Reviewers: While conceptually the program team would be what is utilized, we do not have control over that list and it is out of date. So, need to just list us.